src/instana/__init__.py

```diff
@@ -200,8 +201,8 @@ def boot_agent() -> None:
             storage,  # noqa: F401
         )
         from instana.instrumentation.tornado import (
-            client,  # noqa: F401
-            server,  # noqa: F401
+            client,  # noqa: F401, F811
+            server,  # noqa: F401, F811
```

Comment:
My suggestion is not to prevent this `F811` error to being captured, but solve it. For this, what happens if you give names to the imports? Please, check the code if the following works:

```
        from instana.instrumentation.aiohttp import (
            client as aiohttp_client,  # noqa: F401
            server as aiohttp_server,  # noqa: F401
        )
```
```suggestion
            client as tornado_client,  # noqa: F401
            server as tornado_server,  # noqa: F401
```